src/AzureExtension/DataManager/AzureDataManager.cs

```diff
@@ -262,6 +293,11 @@ public async Task UpdateDataForPullRequestsAsync(AzureUri repositoryUri, string
         return GetPullRequests(repositoryUri.Organization, repositoryUri.Project, repositoryUri.Repository, developerId, view);
     }
 
+    public PullRequests? GetPullRequestsForLoggedInDeveloperIds()
+    {
+        return null;

```

Comment:
null?

Reply:
This was a little future thinking on my part, that method is not used anywhere. I removed it.